tests/neo.UnitTests/SmartContract/UT_ApplicationEngine.cs

```diff
@@ -26,10 +26,16 @@ public void TestBinary()
             var data = new byte[0];
             CollectionAssert.AreEqual(data, engine.Base64Decode(engine.Base64Encode(data)));
 
+            CollectionAssert.AreEqual(data, engine.Base58Decode(engine.Base58Encode(data)));
```

Comment:
Duplicate with L27?

Reply:
The logic is the same but the first tests Base64 and the other tests Base58.